public/app/features/dashboard-scene/serialization/layoutSerializers/utils.ts

```diff
@@ -274,8 +279,9 @@ export function getDataSourceForQuery(querySpecDS: DataSourceRef | undefined | n
 function panelQueryKindToSceneQuery(query: PanelQueryKind): SceneDataQuery {
   return {
     refId: query.spec.refId,
-    datasource: getRuntimePanelDataSource(query.spec.query),
     hide: query.spec.hidden,
+    // If the query has no group, it means it's a default empty query, so we don't need to set a datasource
+    ...(!query.spec.query.group ? {} : { datasource: getRuntimePanelDataSource(query.spec.query) }),
```

Comment:
The query group could be an empty string; this is why we fallback to empty object, because getRuntimePanelDataSource must include a valid group